politeiawww/api/v1/v1.go

```diff
@@ -710,3 +714,42 @@ type UserCommentsVotes struct{}
 type UserCommentsVotesReply struct {
 	CommentsVotes []CommentVote `json:"commentsvotes"`
 }
+
+// VoteOptionResult describes a VotingOption info
+// along with the total votes it received
+type VoteOptionResult struct {
+	Option        VoteOption `json:"option"`        // Vote Option
+	VotesReceived uint64     `json:"votesreceived"` // Votes received by the option
+}
+
+// CastedVotesSummary describes the total votes for a proposal
+// and the VoteOptionResult for each available voting option
+type CastedVotesSummary struct {
+	TotalVotes    int                `json:"totalvotes"`    // Proposal's total votes
+	OptionsResult []VoteOptionResult `json:"optionsresult"` // VoteOptionResult for each option
+}
+
+// VotingStatus describes the voting status for a given proposal
+type VotingStatus struct {
+	Token        string             `json:"token"`                  // Censorship token
+	Status       PropVotingStatus   `json:"status"`                 // Voting status (e.g finished, active, not started)
+	VotesSummary CastedVotesSummary `json:"votessummary,omitempty"` // Summary (won't be provided when fetching all voting status)
+}
+
+// ProposalsVotingStatus is a commmand to fetch the current voting status for
+// all public proposals
+type ProposalsVotingStatus struct{}
+
+// ProposalsVotingStatusReply returns the voting status for all public proposals
+type ProposalsVotingStatusReply struct {
+	ProposalsVoting []VotingStatus `json:"proposalsvoting"` // Public proposal's voting status
+}
+
+// ProposalVotingStatus is a command to fetch the the current voting status
+// for a single public proposal
+type ProposalVotingStatus struct{}
+
+// ProposalVotingStatusReply returns the voting status for a single public proposal
+type ProposalVotingStatusReply struct {
+	ProposalVoting VotingStatus `json:"proposalvoting"`
```

Comment:
It looks like we should not have VotingStatus struct at all since the contents of that struct can be the answer to this command.

Rename all Voting to Vote in these structs.